### ISSUE TITLE
No longer possible to use qalc from scripts?

### ISSUE THREAD

**Issue by mnn** [Qalculate/libqalculate#317]
I always get a long message about dots and it waits for an answer from stdin. My answer is not saved and I didn't see any way (command line argument or environment variable) to set it.

```sh
$ qalc 1.0
Please select interpretation of dots (".").

0 = Both dot and comma as decimal separators (default)
(1.2 = 1,2)

1 = Dot as thousands separator
(1.000.000 = 1000000)

2 = Only dot as decimal separator
(1.2 + root(16, 4) = 3.2)

Dot interpretation: 0
1 = 1
$ qalc 1.0
Please select interpretation of dots (".").

0 = Both dot and comma as decimal separators (default)
(1.2 = 1,2)

1 = Dot as thousands separator
(1.000.000 = 1000000)

2 = Only dot as decimal separator
(1.2 + root(16, 4) = 3.2)

Dot interpretation:
```

This new behaviour has made rofi-calc very annoying to use. https://github.com/svenstaro/rofi-calc/issues/69

**Comment by hanna-kn** [comment]
Workarounds: 
- set ignore dot / nodot to 0 (e.g. `qalc -s "nodot 0" 1.0`)
- set `dot_as_separator=0` in ~/.config/qalc.cfg
- answer the question once in interactive mode (e.g. `qalc -i 1.0`), 

This has now been fixed (in version 3.20.1).

**Comment by mnn** [comment]
It works nicely, thank you for a quick fix :slightly_smiling_face:.
